service/models.py

```diff
@@ -43,37 +49,56 @@ def create(self):
 
     def update(self):
         """
-        Updates a YourResourceModel to the database
+        Updates a Customer to the database
         """
         logger.info("Saving %s", self.name)
+        if not self.id:
+            raise DataValidationError("Update called with empty ID field")
         db.session.commit()
 
     def delete(self):
-        """ Removes a YourResourceModel from the data store """
+        """ Removes a Customer from the data store """
         logger.info("Deleting %s", self.name)
         db.session.delete(self)
         db.session.commit()
 
     def serialize(self):
-        """ Serializes a YourResourceModel into a dictionary """
-        return {"id": self.id, "name": self.name}
+        """ Serializes a Customer into a dictionary """
+        return {"id": self.id, 
+                "name": self.name,
+                "lastname": self.lastname,
+                "email": self.email,
+                "phone": self.phone,
+                "address": self.address,
```

Comment:
As above comment:
This should be split into 2 lines (line1 and line 2) like in the zenhub issue schema description